lighthouse-core/gather/driver.js

```diff
@@ -793,6 +840,13 @@ class Driver {
     // CPU listener. Resolves when the CPU has been idle for cpuQuietThresholdMs after network idle.
     let waitForCPUIdle = this._waitForNothing();
 
+    const waitForSecurityCheck = this._waitForSecurityCheck();
+    const securityCheckPromise = waitForSecurityCheck.promise.then((err) => {
```

Comment:
super nit: but I might have the promise resolve with the explanations and this fn create the error.

that feels a little more mainstream to resolve with data and then the type of the error is clear from within this fn

Reply:
done